doc/install.rst

```diff
@@ -322,6 +321,68 @@ in a special config file $XDG_CONFIG_HOME/looking-glass/imgui.ini.
 Please refrain from editing this file manually as your changes might
 be overwritten by the client.
 
+.. _client_config_widget:
+
+Configuration Widget
+~~~~~~~~~~~~~~~~~~~~
+
+The Configuration Widget is accessible through the Overlay mode. The
+widget has multiple tabs that allow setting a variety of modes and
+parameters for the application.
+
+Settings tab
+^^^^^^^^^^^^
+
+- Performance Metrics: Enabling this will turn on the Metrics Graphs
+  Widget. Multiple graphs can be enabled and they will stack vertically.
+- EGL: Modify EGL features such as the algorithm used for scaling and
+  enabling night vision mode.
+Changes in the Settings tab are not persistent and will change back to
+their default values when the client is restarted.
+
+EGL Filters tab
+^^^^^^^^^^^^^^^
+
+The EGL Filters tab is a GUI for enabling, configuring and ordering
+the post-processing filter stack. Each filter can be expanded to open
+its settings. Filters can also be re-ordered by dragging them up or down.
+Filters application is cumulative, and order is important (e.g. applying CAS
+before FSR might have different results than the reverse). Users are
+encouraged to experiment with the order and parameters to achieve optimal
+results. The currently available filters include:
+
+- Downscaler: Filter for downscaling resolution. It can be used
+  to test how different input resolutions effect the filter stack and
+  find the best stack tuning for the application being used on the guest.
+  The filter has a pixel-size setting that is used to set the effective
+  downscaling ratio and can be set to use different downscaling algorithms.
+- AMD FidelityFX Super Resolution (FSR): Filter implementing  a rendering
+  technique for upscaling resolution. FSR works by accepting the original
+  frames (on the guest) at lower resolution, then applying a spatial upscaling
+  algorithm in the client to make the final result look as though it is
+  rendered in high-resolution. The FSR filter interface allows for fine
+  tuning the sharpness factor of the algorithm and shows the equivalent
+  quality mode based on the ratio between the original frame resolution
+  and the client resolution.
+- AMD FidelityFX Contrast Adaptive Sharpening (CAS): Filter that
+  increases visual quality by applying a sharpening algorithm to the
+  frame. CAS can sometimes restore detail lost in a typical upscaling
+  application. The CAS filter interface has an adjustable sharpness
+  control.
+
+The filter stack settings and order can be saved to presets so that it
+can be conveniently recalled at a later time. As filter settings are
+usually application specific, multiple presets can be defined for each
+case scenario. To save a preset, click on "Save preset as..." and enter
+a preset name. Presets are recalled by selecting them in the "Preset name"
+pull down. Presets are persistent and are stored on disk at
+``$XDG_CONFIG_HOME/looking-glass/presets``.
```

Comment:
Is this too much detail? Do we want users to mess with the preset storage?

Reply:
I can add some text that will discourage users from modifying it, but i still think it's a good idea to detail where LG is storing its magic